Makefile

```diff
@@ -52,6 +53,42 @@ open-project: ## open project on github
 lint: ## check style using tox and flake8 for Python 2 and Python 3
 	$(IN_VENV) tox -e py27-lint && tox -e py34-lint
 
+uwsgi-rebuild-validation: ## rebuild uwsgi_config.yml kwalify schema against latest uwsgi master.
+	$(CONFIG_MANAGE) build_uwsgi_yaml
+
+tool-shed-config-validate: ## validate tool shed YAML configuration file
```

Comment:
``uwsgi-rebuild-validation`` isn't something I'd expect deployers to run that is something we build and ship to them. I guess . a config validate that checked Galaxy, Reports, and Tool Shed configs if they are present would be cool though - I'd hope not too many people have tool shed configurations though 😓.